x-pack/legacy/plugins/siem/public/pages/case/components/user_list/index.tsx

```diff
@@ -59,12 +67,17 @@ const renderUsers = (users: ElasticUser[]) => {
   ));
 };
 
-export const UserList = React.memo(({ headline, users }: UserListProps) => {
+export const UserList = React.memo(({ email, headline, users }: UserListProps) => {
+  const handleSendEmail = useCallback((emailAddress: string | undefined | null) => {
+    if (emailAddress && emailAddress != null) {
+      window.open(`mailto:${emailAddress}?subject=${email.subject}&body=${email.body}`, '_blank');
```

Comment:
Is there a "Kibana" way to do it? Like a component or a utility function.

Reply:
lets ask @XavierM

Reply:
nope, he said to go forward with window.open